lib/accumulate_distribute/events/life_start.js

```diff
@@ -44,7 +44,7 @@ const onLifeStart = async (instance = {}) => {
   }
 
   if (hasIndicatorCap(args)) {
-    const IndicatorClass = HFI[relativeCap.type.toUpperCase()]
+    const IndicatorClass = HFI[relativeOffset.type === 'ma' ? 'SMA' : 'EMA']
```

Comment:
is `relativeOffset` the correct variable in this case? cuz the if stmt is checking for indicator cap

Reply:
no it's not .. fixed the typo.. thanks for pointing it out